resources/aprime_template_vs_obs.bash

```diff
@@ -378,42 +378,41 @@ fi
 
 # LOAD THE MACHINE-SPECIFIC ANACONDA-2.7 ENVIRONMENT
 source $MODULESHOME/init/bash  
-if [ $machname == "edison" ]; then
+if [ ${HOSTNAME:0:3} == "nid" ]; then
   module unload python
   module unload python_base
   source /global/project/projectdirs/acme/software/anaconda_envs/edison/base/etc/profile.d/conda.sh
-  conda activate e3sm_unified_1.2.0_py2.7_nox
+  conda activate e3sm_unified_1.2.3_py2.7_nox
   export NCO_PATH_OVERRIDE=No
-elif [ $machname == "cori" ]; then
+elif [ ${HOSTNAME:0:4} == "cori" ]; then
   module unload python
   module unload python_base
   source /global/project/projectdirs/acme/software/anaconda_envs/cori/base/etc/profile.d/conda.sh
-  conda activate e3sm_unified_1.2.0_py2.7_nox
+  conda activate e3sm_unified_1.2.3_py2.7_nox
   export NCO_PATH_OVERRIDE=No
-elif [ $machname == "rhea" ] || [ $machname == "titan" ]; then
+elif [ ${HOSTNAME:0:4} == "rhea" ] || [ ${HOSTNAME:0:5} == "titan" ]; then
   module unload python
   source /ccs/proj/cli900/sw/rhea/e3sm-unified/base/etc/profile.d/conda.sh
-  conda activate e3sm_unified_1.2.0_py2.7_nox
-  export NCO_PATH_OVERRIDE=No
-elif [ $machname == "acme1" ]; then
-  source /usr/local/e3sm_unified/envs/base/etc/profile.d/conda.sh
-  conda activate e3sm_unified_1.2.0_py2.7_nox
+  conda activate e3sm_unified_1.2.3_py2.7_nox
   export NCO_PATH_OVERRIDE=No
-elif [ $machname == "aims4" ]; then
+elif [ ${HOSTNAME:0:5} == "aims4" ] || [ ${HOSTNAME:0:5} == "acme1" ]; then
   source /usr/local/e3sm_unified/envs/base/etc/profile.d/conda.sh
-  conda activate e3sm_unified_1.2.0_py2.7_nox
+  conda activate e3sm_unified_1.2.3_py2.7_nox
   export NCO_PATH_OVERRIDE=No
-elif [ $machname == "lanl" ]; then
+elif [ ${HOSTNAME:0:4} == "wolf" ] || [ ${HOSTNAME:0:7} == "grizzly" ]; then
```

Comment:
these host names are "wf-fe*" and "gr-fe*", though I don't think it's a big concern because I doubt processflow will be used there.